boa3/analyser/moduleanalyser.py

```diff
@@ -87,6 +88,19 @@ def __include_method(self, method_id: str, method: Method):
         if method_id not in self.__current_module.symbols:
             self.__current_module.include_method(method_id, method)
 
+    def get_symbol(self, symbol_id: str) -> Optional[ISymbol]:
+        if symbol_id in self.__current_scope.symbols:
+            # the symbol exists in the local scope
+            return self.__current_scope.symbols[symbol_id]
+        elif symbol_id in self.__current_module.symbols:
+            # the symbol exists in the module scope
+            return self.__current_module.symbols[symbol_id]
```

Comment:
Just a reminder you told me this is wrong

Reply:
It was wrong here actually
https://github.com/CityOfZion/neo3-boa/blob/711b7ad42aeddb3b4bc5244ae5784241a6063301/boa3/analyser/typeanalyser.py#L66-L69